### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by mikenairn**
## Motivation

Enable building the app via the Mobile CI/CD service.

## Description

Adds a Jenkinsfile

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/**
+* Android Jenkinsfile
+*/
+node("android"){
+  stage("Checkout"){
+    checkout scm
+  }
+
+  stage ("Prepare"){
+    if(params.FH_CONFIG_CONTENT) {
```
Comment (odra) [inline Jenkinsfile]: We can remove the FH_CONFIG thing and move the command `sh 'chmod +x ./gradlew'` from the build stage to this one.

**Comment by maleck13** [comment]
@wei-lee ok to get someone to review this

**Comment by secondsun** [comment]
How do we verify this?

**Comment by mikenairn** [comment]
> How do we verify this?

@secondsun Good question, not easily unless you have the latest version of the Mobile CI/CD service running in your local openshift and have all the ongoing mobile UI openshift changes.

As it is, it doesn't work anyway, at least not for both release and debug builds. This appears to be down to the way this project is setup, and in particular how we are specifying SNAPSHOT versions of dependencies for some builds (release vs debug). You any idea if this is something you plan on changing, as we will need to figure out how we can do both debug and release builds from the same source.

**Comment by PhilipGough** [comment]
Some problems with building a release version here, see https://issues.jboss.org/browse/AEROGEAR-2913

**Comment by maleck13** [comment]
link https://github.com/aerogear/android-showcase-template/pull/33 updated to remove the sign step. There is an associated console change

**Comment by secondsun** [comment]
Does #33 obsolete this?

**Pull request closed without merging**